persistence/src/main/java/org/mskcc/smile/persistence/neo4j/TempoRepository.java

```diff
@@ -132,4 +132,12 @@ void updateTempoAccessLevelBySamplePrimaryIds(@Param("samplePrimaryIds") List<St
             + "WHERE result.primaryId = $primaryId "
             + "RETURN result")
     Map<String, Object> findTempoSampleDataBySamplePrimaryId(@Param("primaryId") String primaryId);
+
+    @Query("MATCH (t: Tempo{smileTempoId: $smileTempoId}) "
+            + "SET t.initialPipelineRunDate = $initialPipelineRunDate, "
+            + "t.embargoDate = $embargoDate, t.accessLevel = $accessLevel")
+    void updateTempoData(@Param("smileTempoId") UUID smileTempoId,
+            @Param("initialPipelineRunDate") String initialPipelineRunDate,
+            @Param("embargoDate") String embargoDate,
+            @Param("accessLevel") String accessLevel);
```

Comment:
passing along a populated `Tempo` node resulted in a recursive/stack overflow error. Passing along the explicit values to update and find record on prevents that issue from happening.